parts/linux/cloud-init/artifacts/validate-kubelet-credentials.sh

```diff
@@ -40,27 +40,16 @@ logs_to_events() {
     fi
 }
 
-validateBootstrapKubeconfig() {
+validate() {
     local kubeconfig_path=$1
+    
+    # extract the bootstrap token directly from bootstrap-kubeconfig since we can't use LoadCredential (not supported on 1804)
```

Comment:
why do we care about 1804 ?

Reply:
at the moment we're still building 1804 VHDs and bootstrapping 1804 nodes, so we kinda still need to deal with it - once it's completely removed/deprecated we could come back to this